.rubocop.yml

```diff
@@ -226,3 +226,7 @@ Style/StringLiterals:
 Style/SymbolArray:
   Exclude:
     - 'Rakefile' # because [:spec, :rubocop] isn't a big deal
+
+Style/TernaryParentheses:
+  Exclude:
+    - spec/lib/active_record_utils_spec.rb # ternary expression contains a little math, reads better with parens
```

Comment:
don't need to bother changing, but fine to shorten comment to `# reads better with parens`

Reply:
also fixed this (and the lack of line break) at the end